src/App.jsx

```diff
@@ -1,23 +1,41 @@
-import logo from "./logo.svg";
 import styles from "./App.module.scss";
+import { Routes, Route } from "solid-app-router";
+import { createEffect } from "solid-js";
+import { loginUrl } from "Constants/deriv-urls";
+import Endpoint from "Routes/endpoint";
```

Comment:
not sure if the PR is ready to be reviewed, but can we use lazy loading @farrah-deriv